### PR TITLE
digid extend attributes

### PR DESCRIPTION
**Pull request by KaushikGondaliya**

### CODE REVIEW

**Comment by jane-doe** [comment]
Can one of the admins verify this patch?

**Review by viteke — APPROVED** [review]
Ok

File: SGO/sgo/attributes/gender.ttl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:gender
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "gender";
+	dcterms:description """person's gender(sex). i.e male, female..""";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
```
Comment (cy303) [inline SGO/sgo/attributes/gender.ttl:11]: I don't think timing intervals are of much use here, I would remove that comment

File: SGO/sgo/attributes/locationPrecision.ttl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:locationPrecision
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "locationPrecision";
+	dcterms:description """Estimated precision (in meters) of location.""";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
```
Comment (cy303) [inline SGO/sgo/attributes/locationPrecision.ttl:11]: precision is also not a time interval

File: SGO/sgo/attributes/yearOfBirth.ttl [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:yearOfBirth
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "yearOfBirth";
+	dcterms:description """Birth year of person.""";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
```
Comment (cy303) [inline SGO/sgo/attributes/yearOfBirth.ttl:11]: and a year isn't a period but a fixed point in time as well...

**Review by cy303 — CHANGES_REQUESTED** [review]

**Pull request merged**